src/components/Reports/ReportsPageRanking/ReportsPageRanking.js

```diff
@@ -79,23 +86,48 @@ class ReportsPageRanking extends React.Component {
               </small>
 
               {pages.map((item, index) => (
-                <div key={index} className="page-ranking--item">
-                  <p className="text-ranking">
-                    <strong>{index + 1}</strong>
-                  </p>
-                  <a
-                    className="link-ranking"
-                    href={item.url}
-                    target="_blank"
-                    rel="noopener noreferrer"
-                  >
-                    {item.name}
-                  </a>
-                  <p className="text-ranking">
-                    <strong>{item.totalVisitors}</strong>{' '}
-                    <FormattedMessage id="reports_pageranking.total_visits" />
-                  </p>
-                </div>
+                <PageRankingItem key={index}>
+                  <div>
+                    <PageRankingItemText>
+                      <strong>{index + 1}</strong>
```

Comment:
What do you think about moving `<strong>` element inside `PageRankingItemText`?

Something like:

```html
<PageRankingItemText index={index} />
```

Reply:
Oh, this component is here only for styles. What do you think about converting it in something with better semantic? like

```html
<PageRankingItem
    index={index}
    item={item} />
```

Reply:
i understand the structure better is in the same place. like this:
![image](https://user-images.githubusercontent.com/8861133/64177207-671c3080-ce35-11e9-9e30-54840bfb1da8.png)

what do you think @cbernat ?

Reply:
In this way, the following changes will be only in `PageRankingItem`

Reply:
Nice example @fcoronelmakingsense !

Maybe `PageRankingItemText` is not necessary anymore if you add `PageRankingItem`

Reply:
`PageRankingItem` is a styled component, I'm not so sure about how to combine this styles with logic, all over internet I've seen them separated. Perhaps @fcoronelmakingsense you could dive deeper in this matter, see if it can be done or not.